extensions/wikia/SkinChooser/SkinChooser.class.php

```diff
@@ -250,7 +250,7 @@ public static function onGetSkin( RequestContext $context, &$skin ) {
 				} else {
 					$userSkin = 'oasis';
 				}
-			} else if ( !empty( $wgAdminSkin ) && $userSkin != 'oasis' && $userSkin != 'monobook' && $userSkin != 'wowwiki' && $userSkin != 'lostbook' ) {
+			} else if ( !empty( $wgAdminSkin ) && $userSkin != 'oasis' && $userSkin != 'monobook' ) {
```

Comment:
why did you remove `lostbook` check?

Reply:
there's no such skin on our site, it refers to some product of a bygone era